shipstation_integration/shipments.py

```diff
@@ -158,3 +156,35 @@ def create_delivery_note(shipment, sales_invoice):
 	dn.submit()
 	frappe.db.commit()
 	return dn
+
+def create_shipment(shipment, delivery_note, store):
+	shipment_doc: "Shipment" = make_shipment(delivery_note.name)
```

Comment:
If you're adding the "Shipment" typing, you'll have to import it. Check how the "DeliveryNote" class is being imported.